### PR TITLE
run command geneva action

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
MSFT: provides run command geneva action for kubelet and networkmanager restart
```

Docker restart not working properly so not added - follow up in #1256

### CODE REVIEW

**Comment by jim-minter** [comment]
@charlesakalugwu ptal
/hold
because it seems that `docker restart` irrevocably breaks our VMs at the moment?  :-(

File: pkg/cluster/admin_actions.go [hunk 1]
```diff
@@ -31,3 +34,10 @@ func (u *simpleUpgrader) ListVMHostnames(ctx context.Context, cs *api.OpenShiftM
 
 	return hostnames, nil
 }
+
+func (u *simpleUpgrader) RunCommand(ctx context.Context, cs *api.OpenShiftManagedCluster, scaleset, instanceID, command string) error {
+	return u.vmc.RunCommand(ctx, cs.Properties.AzProfile.ResourceGroup, scaleset, instanceID, compute.RunCommandInput{
+		CommandID: to.StringPtr("RunShellScript"),
```
Comment (thekad) [inline pkg/cluster/admin_actions.go]: nit: `RunAdminShellScript`

File: pkg/fakerp/admin_handlers.go [hunk 2]
```diff
@@ -173,3 +175,29 @@ func (s *Server) handleForceUpdate(w http.ResponseWriter, req *http.Request) {
...
+		return
+	}
+
+	err = s.plugin.RunCommand(ctx, cs, hostname, api.Command(command))
+	if err != nil {
+		s.internalError(w, fmt.Sprintf("Failed to run command: %v", err))
+		return
+	}
+
+	s.log.Infof("ran command on %s", hostname)
```
Comment (thekad) [inline pkg/fakerp/admin_handlers.go]: ran _what_ command?

Comment (thekad) [reply pkg/fakerp/admin_handlers.go]: also: would it be useful to at least log how long the command took _in lieu_ of actual metrics for now?

File: pkg/cluster/admin_actions.go (hunk 1 above)
Comment (jim-minter) [reply pkg/cluster/admin_actions.go]: I don't think so - this is an Azure-defined constant: see `az vm run-command list --location eastus` for the available ones

File: pkg/fakerp/admin_handlers.go (hunk 2 above)
Comment (jim-minter) [reply pkg/fakerp/admin_handlers.go]: both fixed, thanks - command timing can be seen from log timestamps

File: pkg/plugin/plugin.go [hunk 3]
```diff
@@ -356,3 +356,33 @@ func (p *plugin) BackupEtcdCluster(ctx context.Context, oc *api.OpenShiftManaged
...
+
+func (p *plugin) RunCommand(ctx context.Context, oc *api.OpenShiftManagedCluster, hostname string, command api.Command) error {
+	if !validate.IsValidAgentPoolHostname(hostname) {
+		return fmt.Errorf("invalid hostname %q", hostname)
+	}
+
+	var script string
+	switch command {
+	case api.CommandRestartNetworkManager:
+		script = "systemctl restart NetworkManager.service"
```
Comment (thekad) [inline pkg/plugin/plugin.go]: nit: should we point to the absolute path of the command? not sure if Geneva commands will be affected by path changes or etc...

**Comment by thekad** [comment]
non blocking nit, thanks!
/lgtm

**Pull request merged**